src/pages/reference/handlers/graphql.md

```diff
@@ -224,6 +224,7 @@ other options will be ignored and the schema exported from the file will be used
 -  `operationHeaders` (type: `JSON`) - JSON object representing the Headers to add to the runtime of the API calls only for operation during runtime
 -  `useGETForQueries` (type: `Boolean`) - Use HTTP GET for Query operations
 -  `method` (type: `String (GET | POST)`) - HTTP method used for GraphQL operations
+-  `subscriptionsEndpoint` (type: `String`) - A URL to your endpoint serving all subscription queries for this source
```

Comment:
we are blocking this field internally 'subscriptionsEndpoint' we can remove this